Source/Core/DolphinQt/Debugger/AssemblerWidget.cpp

```diff
@@ -701,21 +701,23 @@ void AssemblerWidget::OnTabChange(int index)
 QString AssemblerWidget::TabTextForEditor(AsmEditor* editor, bool with_dirty)
 {
   ASSERT(editor != nullptr);
-  QString dirtyFlag = QStringLiteral();
-  if (editor->IsDirty() && with_dirty)
-  {
-    dirtyFlag = QStringLiteral(" *");
-  }
 
-  if (editor->Path().isEmpty())
+  QString result;
+  if (!editor->Path().isEmpty())
+    result = editor->EditorTitle();
+  else if (editor->EditorNum() != 0)
+    result = tr("New File");
+  else
+    result = tr("New File (%1)").arg(editor->EditorNum() + 1);
```

Comment:
At a glance, this seems backwards? Or the conditional should be == 0.

Reply:
Yes. I was considering flipping the order of the two paths, then decided against it, and apparently forgot to restore the condition.